meson.build

```diff
@@ -23,8 +24,18 @@ libavutil = dependency('libavutil', version: '>= 56.70.100')
 libswresample = dependency('libswresample', version: '>= 3.9.100')
 libswscale = dependency('libswscale', version: '>= 5.9.100')
 
-libplacebo = dependency('libplacebo', version: '>=6.338.0', fallback: ['libplacebo', 'libplacebo'],
-    default_options: ['default_library=static', 'demos=false'])
```

Comment:
What if we add `libplacebo-wrap` disabled by default and force users to explicitly enable it, if they want to.

``` meson
if get_option('libplacebo-wrap')
    libplacebo = dependency('libplacebo', version: '>=6.338.0', fallback: ['libplacebo', 'libplacebo'],
        default_options: ['default_library=static', 'demos=false'])
else
    libplacebo = dependency('libplacebo', version: '>=6.338.0')
endif
```

Reply:
We still show the standard dependency error with this PR, but doing it this way works too. I don't have a preference but if a `libplacebo-wrap` option is preferred then I can change it to that. Though at that point I'm not sure why we are bothering to use wraps instead of git submodules